src/parser.rs

```diff
@@ -918,17 +906,26 @@ impl<'a> Parser<'a> {
         }
     }
 
-    /// Parses struct fields
+    /// Parses struct field comment
     ///
-    /// `line` the Solidity definition of the struct field
+    /// `comment_raw` the Solidity definition of
+    /// the struct field multiline or one line comment
     ///
-    /// returns the struct field as `StructField` struct
-    fn parse_struct_field(&mut self, line: String) -> StructField {
-        let tokens = split(&line, " ", None);
-        let field_type = self.convert_variable_type(tokens[0].to_owned());
-        let mut name = tokens[1].to_owned();
-        name.remove_matches(";");
-        StructField { name, field_type }
+    /// returns the comment as String
+    fn parse_struct_field_comment(&mut self, mut comment_raw: String) -> String {
+        if comment_raw.is_empty() {
+            comment_raw
+        } else if comment_raw.contains("/*") {
+            comment_raw.remove_matches("/*");
```

Comment:
we want to get rid of remove matches, you can add those characters to the regex

Reply:
if we add it to regex, we will need to 2 comments groups between field and 2 groups after. For instance:
`"(?P<comment1>(\n\s*//.*)*|(\n\s*/\*(.*\n)*?.*\*/\s*))? ..."`
in this case we need to move /* before group start, but in this case it won't work with one line comments group. As result we need to create separate one. It equal to add one more function call in parser code. Therefore I think will be better to leave it as it is. 
What do you think?

Reply:
ok leave it